source/brailleDisplayDrivers/superBrl.py

```diff
@@ -84,22 +79,16 @@ def __init__(self,port="Auto"):
 			if self.numCells:
 				# ok, it is a SuperBraille
 				log.info("Found superBraille device, version %s"%self.version)
-				found = True
 				break
+			else:
+				self._dev.close()
 		else:
-			self._dev.close()
-		if not found:
-			raise RuntimeError, "No SuperBralle found"
+			raise RuntimeError, "No SuperBraille found"

```

Comment:
Sorry, missed this. This is old Python syntax. Please change to:

`raise RuntimeError("No SuperBraille found")`

That is, pass the message to the exception's constructor.